.travis.yml

```diff
@@ -34,10 +34,6 @@ jobs:
           os: windows
           script:
               - npm run test:scripts -- $SINCE_ARG
-        - name: "Script tests (Node 10, linux)"
```

Comment:
I'm afraid of removing this test we could literally break users with Node 10 + Windows

Reply:
How? We test Node 10 + windows. And Node latest + windows; and node latest + linux. I have hard times imagining this forth combination will discover any real issues.

Reply:
ohhh, true :P